mavros_msgs/msg/ObstacleAvoidance.msg

```diff
@@ -0,0 +1,18 @@
+# MAVLink message: OBSTACLE_AVOIDANCE
+# http://mavlink.org/messages/common#OBSTACLE_AVOIDANCE
+
+std_msgs/Header header
+
+uint8 type # See enum MAV_TRAJECTORY_REPRESENTATION.
+uint8 MAV_TRAJECTORY_REPRESENTATION_WAYPOINTS = 0
+uint8 MAV_TRAJECTORY_REPRESENTATION_BEZIER = 1
+
+mavros_msgs/PositionTarget point_1
+mavros_msgs/PositionTarget point_2
+mavros_msgs/PositionTarget point_3
+mavros_msgs/PositionTarget point_4
+mavros_msgs/PositionTarget point_5
```

Comment:
Why not:
```
mavros_msgs/PositionTarget[5] points
```

Reply:
I think is help with readability to have a bit more text and less indexes.

Reply:
Just my 2 cents - I prefer the array.

Reply:
Actuallly @mrivi I think it would fit better also.